pkg/cloudprovider/openstack/openstack.go

```diff
@@ -515,7 +515,7 @@ func (lb *LoadBalancer) CreateTCPLoadBalancer(name, region string, externalIP ne
 
 		_, err = members.Create(lb.network, members.CreateOpts{
 			PoolID:       pool.ID,
-			ProtocolPort: port,
+			ProtocolPort: ports[0], //FIXME: need to handle multi-port
```

Comment:
There is no multiport feature in OpenStack LBaaS - so we'll need to create a new VIP with a new Pool (and new Pool Members) for each port.  Basically just put a loop around this function body.

Urgh, no it isn't that easy because then you won't have a 1:1 with k8s' LoadBalancer `name` and the OpenStack VIPs, which will break the other `TCPLoadBalancer` methods.  Assign me a followon bug and I'll have to come up with something involving regexes and non-atomic operations :(

It looks to me like the GCE change above for this implements a port *range* - is this the intended multiport semantics, or just an easy implementation?  (neither interpretation exists in OpenStack, just wondering which one I should implement)